dbconnect.py

```diff
@@ -0,0 +1,72 @@
+#TODO Needs more work to make it complete
+
+import psycopg2
+
+
+class DBConnect(object):
+    """
+    Class for connecting to a postgresql database using a with statement
+    """
+
+    def __init__(self, dbhost='', db='', dbuser='', dbpass='', dbport=3306, autocommit=False,
+                 *args, **kwargs):
+        try:
+            self.conn = psycopg2.connect(host=dbhost, database=db, user=dbuser,
```

Comment:
There is by wrapping it in another manager, but that was one of the purposes of this class. The __enter__ and __exit__ methods should provide the close-outs.